### PR TITLE
Export TypeScript types for colors and modifiers

### PR DESCRIPTION
**Pull request by SimenB**
Closes #336

### CODE REVIEW

**Comment by SimenB** [comment]
Should I open up a separate PR for https://github.com/chalk/chalk/tree/v2-release?

File: index.d.ts [hunk 1]
```diff
@@ -20,6 +20,71 @@ declare const enum LevelEnum {
 	TrueColor = 3
 }
 
+/**
+Available foreground colors.
```
[the remark this answers is not in the document]
Comment (SimenB) [reply index.d.ts]: Should I word that comment differently, or add more to the type?

[the remark this answers is not in the document]
Comment (sindresorhus) [reply index.d.ts]: Just make it clear that it's not the only available colors.

**Comment by sindresorhus** [comment]
> Should I open up a separate PR for v2-release ?

Nah, I intend to get Chalk 3 out in the next few weeks.

File: index.d.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (SimenB) [reply index.d.ts]: Went with

>Available builtin foreground colors. Other colors are also available: https://github.com/chalk/chalk/blob/master/readme.md#256-and-truecolor-color-support

Is that ok or do you want something else?

[the remark this answers is not in the document]
Comment (SimenB) [reply index.d.ts]: I've pushed it, but GH is having issues: https://www.githubstatus.com/

Diff:

```diff
diff --git c/index.d.ts w/index.d.ts
index afd382c..81b6efc 100644
--- c/index.d.ts
+++ w/index.d.ts
@@ -21,50 +21,50 @@ declare const enum LevelEnum {
 }
 
 /**
-Available foreground colors.
+Available builtin foreground colors. Other colors are also available: https://github.com/chalk/chalk/blob/master/readme.md#256-and-truecolor-color-support
 */
 declare type ForegroundColor =
-	| "black"
-	| "red"
-	| "green"
-	| "yellow"
-	| "blue"
-	| "magenta"
-	| "cyan"
-	| "white"
-	| "gray"
-	| "grey"
-	| "blackBright"
-	| "redBright"
-	| "greenBright"
-	| "yellowBright"
-	| "blueBright"
-	| "magentaBright"
-	| "cyanBright"
-	| "whiteBright";
+	| 'black'
+	| 'red'
+	| 'green'
+	| 'yellow'
+	| 'blue'
+	| 'magenta'
+	| 'cyan'
+	| 'white'
+	| 'gray'
+	| 'grey'
+	| 'blackBright'
+	| 'redBright'
+	| 'greenBright'
+	| 'yellowBright'
+	| 'blueBright'
+	| 'magentaBright'
+	| 'cyanBright'
+	| 'whiteBright';
 
 /**
-Available background colors.
+Available builtin background colors. Other colors are also available: https://github.com/chalk/chalk/blob/master/readme.md#256-and-truecolor-color-support
 */
 declare type BackgroundColor =
-	| "bgBlack"
-	| "bgRed"
-	| "bgGreen"
-	| "bgYellow"
-	| "bgBlue"
-	| "bgMagenta"
-	| "bgCyan"
-	| "bgWhite"
-	| "bgGray"
-	| "bgGrey"
-	| "bgBlackBright"
-	| "bgRedBright"
-	| "bgGreenBright"
-	| "bgYellowBright"
-	| "bgBlueBright"
-	| "bgMagentaBright"
-	| "bgCyanBright"
-	| "bgWhiteBright";
+	| 'bgBlack'
+	| 'bgRed'
+	| 'bgGreen'
+	| 'bgYellow'
+	| 'bgBlue'
+	| 'bgMagenta'
+	| 'bgCyan'
+	| 'bgWhite'
+	| 'bgGray'
+	| 'bgGrey'
+	| 'bgBlackBright'
+	| 'bgRedBright'
+	| 'bgGreenBright'
+	| 'bgYellowBright'
+	| 'bgBlueBright'
+	| 'bgMagentaBright'
+	| 'bgCyanBright'
+	| 'bgWhiteBright';
 
 /**
 Available colors.
@@ -75,15 +75,15 @@ declare type Color = ForegroundColor | BackgroundColor;
 Available modifiers.
 */
 declare type Modifiers =
-	| "reset"
-	| "bold"
-	| "dim"
-	| "italic"
-	| "underline"
-	| "inverse"
-	| "hidden"
-	| "strikethrough"
-	| "visible";
+	| 'reset'
+	| 'bold'
+	| 'dim'
+	| 'italic'
+	| 'underline'
+	| 'inverse'
+	| 'hidden'
+	| 'strikethrough'
+	| 'visible';
 
 declare namespace chalk {
 	type Level = LevelEnum;

```

**Comment by sindresorhus** [comment]
Thanks :)

File: index.d.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (SimenB) [reply index.d.ts]: Thanks for dealing with it!

**Pull request merged**